test/functional/ui/float_spec.lua

```diff
@@ -9518,6 +9518,116 @@ describe('float window', function()
         })
       end
     end)
+
```

Comment:
also need a single "validation" tests that runs through all the valid values.

This also somewhat mitigates the fact that the values are repeated at `p_winbd_values` and [`parse_border_style`](https://github.com/neovim/neovim/blob/ac7e0ff32ff18be69e2469ff7fc66370be20a6e1/src/nvim/api/win_config.c#L933-L945)

```lua
    it('#winborder validation', function()
      command('set winborder=single')
      command('set winborder=double')
      command('set winborder=...')
    end)